### PR TITLE
[DO-1699]Make first declarations models

### PR DESCRIPTION
**Pull request by IegorSoboliev**

### CODE REVIEW

**Comment by IegorSoboliev** [comment]
please, take a look at first (but not last))) commit

File: business_register/models/declaration_models.py [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+        db_index=True
+    )
+    nacp_declarant_id = models.PositiveIntegerField(
+        _('NACP id of the declarant'),
+        unique=True,
+        db_index=True
+    )
+    pep = models.ForeignKey(
+        Pep,
+        on_delete=models.CASCADE,
```
Comment (roman-tiukh) [inline business_register/models/declaration_models.py:104]: maybe PROTECT?

File: business_register/models/declaration_models.py [hunk 2]
```diff
@@ -0,0 +1,155 @@
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+from data_ocean.models import DataOceanModel
+from business_register.models.pep_models import Pep
+from location_register.models.ratu_models import RatuCity
+
+
+# ToDo: compare with rules
+
+class Right(DataOceanModel):
```
Comment (roman-tiukh) [inline business_register/models/declaration_models.py:10]: Right?

File: business_register/models/declaration_models.py [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+
+    )
+    city = models.ForeignKey(
+        RatuCity,
+        on_delete=models.CASCADE,
+        related_name='declared_real_estates',
+        verbose_name=_('address'),
+        help_text=_('city where the property is located')
+    )
+    rights = models.ForeignKey(
```
Comment (roman-tiukh) [inline business_register/models/declaration_models.py:65]: rights?

File: business_register/models/declaration_models.py [hunk 4]
```diff
@@ -0,0 +1,155 @@
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
```
[the remark this answers is not in the document]
Comment (IegorSoboliev) [reply business_register/models/declaration_models.py]: changed

File: business_register/models/declaration_models.py [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+    )
+    type = models.CharField(
+        _('type of declaration'),
+        choices=DECLARATION_TYPES,
+        max_length=5
+    )
+    year = models.PositiveIntegerField(
+        _('year of the declaration'),
+        help_text=_('year of the declaration')
+    )
```
[the remark this answers is not in the document]
Comment (IegorSoboliev) [reply business_register/models/declaration_models.py]: changed

File: business_register/models/declaration_models.py (hunk 1 above)
Comment (IegorSoboliev) [reply business_register/models/declaration_models.py]: changed

File: business_register/models/declaration_models.py [hunk 6]
```diff
@@ -0,0 +1,634 @@
...
+        _('NACP id of the declarant'),
+        unique=True,
+        db_index=True
+    )
+    pep = models.ForeignKey(
+        Pep,
+        on_delete=models.PROTECT,
+        related_name='declarations',
+        verbose_name=_('PEP that declares'),
+        help_text=_('politically exposed person that declares')
```
Comment (criticus) [inline business_register/models/declaration_models.py:48]: that -> who

File: business_register/models/declaration_models.py [hunk 7]
```diff
@@ -0,0 +1,634 @@
...
+    )
+    partner = models.ForeignKey(
+        Pep,
+        null=True,
+        blank=True,
+        default=None,
+        on_delete=models.PROTECT,
+        related_name='partner',
+        verbose_name=_('partner'),
+        help_text=_('wife or husband from declaration')
```
Comment (criticus) [inline business_register/models/declaration_models.py:88]: spouse of the declarant

File: business_register/models/declaration_models.py [hunk 8]
```diff
@@ -0,0 +1,634 @@
...
+    )
+    bank_creditor = models.ForeignKey(
+        Company,
+        on_delete=models.PROTECT,
+        related_name='bank',
+        verbose_name=_('bank'),
+        null=True,
+        blank=True,
+        default=None,
+        help_text=_('bank or company that gave the loan')
```
Comment (criticus) [inline business_register/models/declaration_models.py:136]: bank  or company to whom money is owed

File: business_register/models/declaration_models.py [hunk 9]
```diff
@@ -0,0 +1,634 @@
...
+    )
+    pep_creditor = models.ForeignKey(
+        Pep,
+        on_delete=models.PROTECT,
+        related_name='creditors',
+        verbose_name=_('PEP-creditor'),
+        null=True,
+        blank=True,
+        default=None,
+        help_text=_('politically exposed person that gave the loan')
```
Comment (criticus) [inline business_register/models/declaration_models.py:146]: to whom money is owed

File: business_register/models/declaration_models.py (hunk 6 above)
Comment (IegorSoboliev) [reply business_register/models/declaration_models.py:48]: thanks)

File: business_register/models/declaration_models.py (hunk 8 above)
Comment (IegorSoboliev) [reply business_register/models/declaration_models.py:136]: changed

File: business_register/models/declaration_models.py (hunk 9 above)
Comment (IegorSoboliev) [reply business_register/models/declaration_models.py:146]: changed

**Review by roman-tiukh — APPROVED** [review]

**Pull request merged**